dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -726,6 +734,13 @@ public Token addOrcidToken(@ApiParam(hidden = true) @Parameter(hidden = true, na
             token.setRefreshToken(refreshToken);
             token.setUserId(user.getId());
             token.setUsername(username);
+            TokenScope tokenScope = TokenScope.getEnumByString(scope);
```

Comment:
I went down this rabbit hole since I was only familiar with scopes in JWT. According to https://tools.ietf.org/html/rfc6749#section-3.3, the scope can be a space delimited list of strings. If our use case of ORCID always returns 1 string and we're not using this for any other providers, we can cross that bridge when/if we get there, but wanted to point it out.